packages/core/src/components/containers/refine/index.tsx

```diff
@@ -49,6 +51,7 @@ export interface RefineProps {
     routerProvider: IRouterProvider;
     notificationProvider?: NotificationProvider;
     accessControlProvider?: AccessControlProvider;
+    auditLogProvider?: IAuditLogContext;
```

Comment:
```suggestion
    auditLogProvider?: AuditLogProvider;
```
I think, we should use it with the type in `packages/core/src/contexts/auditLog/IAuditLogContext.ts` file.